.github/workflows/ci.yaml

```diff
@@ -40,10 +40,25 @@ jobs:
           command: lint
           config: .github/ct.yaml
 
+  kubeval-chart:
+    runs-on: ubuntu-latest
+    needs: check-for-chart-changes
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set user permissions
+        run: chmod -R 777 /home/runner/work
+      - name: Run kubeval
+        uses: docker://kiwigrid/gcloud-kubectl-helm
```

Comment:
Could you explain what this is doing?

Reply:
> Could you explain what this is doing?

checks out repo
set file permissions whats a workaround for wrong /home dir in github actions
uses kiwigrids gcloud-kubectl-helm docker image which contains kubeval binary to validate the helm template

Reply:
I would like to avoid adding additional container images. Especially as this isn't versioned.

Why not pull in the binary from https://github.com/instrumenta/kubeval/releases with `curl` instead?

Reply:
because its not only the kubeval binary you need.
helm binary is also needed and has to be initialised.
added the docker tag now.

Reply:
So to me this sounds more like something we should then add to the packages project. As it wouldn't be possible for IoT Packages committers to make any changes to this docker image in the future.

Reply:
i've changed the pr to not use docker.